app/controllers/nwbib/Classification.java

```diff
@@ -369,7 +370,7 @@ private static boolean isItem(JsonNode json, String broaderId) {
 			Map<String, List<JsonNode>> subClasses) {
 		List<String> ids = new ArrayList<>(subClasses.keySet());
 		Collections.sort(ids, Comparator.comparingInt(s -> Integer
-				.parseInt(s.substring("http://www.wikidata.org/entity/Q".length()))));
+				.parseInt(s.substring("https://nwbib.de/spatial#Q".length()))));
 		for (int i = 0; i < ids.size(); i++) {
```

Comment:
"https://nwbib.de/spatial#Q" is also used multiple times and could be a static final.